pkg/apis/unikorn/v1alpha1/types.go

```diff
@@ -731,3 +731,54 @@ type OpenstackServerSpec struct {
 
 type OpenstackServerStatus struct {
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type ServerStorageList struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Items             []ServerStorage `json:"items"`
+}
+
+// schristoff:  StorageSpec
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope=Namespaced,categories=unikorn
+// +kubebuilder:printcolumn:name="age",type="date",JSONPath=".metadata.creationTimestamp"
+type ServerStorage struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ServerStorageSpec    `json:"spec"`
+	Request           ServerStorageRequest `json:"request"`
+	// ??
+	TenantID string
```

Comment:
We need some special information about who's storage we need to be requesting
I think it goes in the Request though